ui-admin/src/study/participants/participantList/search/SearchCriteriaView.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { FacetValue, facetNameAndValueString } from '../../../../api/enrolleeSearch'
+
+import Chip from '@mui/material/Chip'
+import Stack from '@mui/material/Stack'
+
+/**
+ * Provides a view of the current search criteria showing the facets and values that have been selected,
+ * and allowing the user to delete criteria.
+ */
+const SearchCriteriaView = ({ facetValues, updateFacetValues }:
+                               { facetValues: FacetValue[],
+                                 updateFacetValues: (values: FacetValue[]) => void,
+                               }) => {
+  const handleDelete = (deleteFacetValue: FacetValue) => {
+    updateFacetValues(facetValues.filter(facetValue =>
+      facetValue.facet.keyName !== deleteFacetValue.facet.keyName))
+  }
+
+  return (
+    <Stack direction="row" spacing={1}>
+      {facetValues.filter(facetValue => !facetValue.isDefault()).map(facetValue => {
+        return (
+          <Chip
+            key={facetValue.facet.keyName}
+            label={facetNameAndValueString(facetValue)}
+            onDelete={() => handleDelete(facetValue)}
+          />
+        )
+      })}
+    </Stack>
+  )
+}
+
+export default SearchCriteriaView
```

Comment:
I'm hesitant to bring in Material UI just for chips and stacks, especially since we already use Bootstrap. I'm mostly worried about style conflicts and the longer term maintenance burden of having to keep up with both frameworks in our application (especially since we'll probably be tempted to use other Material UI components if they're already within reach).

Something like this would accomplish a similar thing without bringing in Material UI. I know it could still use some style tweaks, but what do you think?


```suggestion
import React from 'react'
import { FacetValue, facetNameAndValueString } from 'api/enrolleeSearch'
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
import { faTimes } from '@fortawesome/free-solid-svg-icons'

/**
 * Provides a view of the current search criteria showing the facets and values that have been selected,
 * and allowing the user to delete criteria.
 */
const SearchCriteriaView = ({ facetValues, updateFacetValues }:
                              { facetValues: FacetValue[], updateFacetValues: (values: FacetValue[]) => void }) => {
  const handleDelete = (deleteFacetValue: FacetValue) => {
    updateFacetValues(facetValues.filter(facetValue =>
      facetValue.facet.keyName !== deleteFacetValue.facet.keyName))
  }

  return (
    <div className="d-flex flex-wrap gap-2">
      {facetValues.filter(facetValue => !facetValue.isDefault()).map(facetValue => {
        return (
          <button
            key={facetValue.facet.keyName}
            className="btn btn-outline-secondary btn-sm rounded-pill"
            onClick={() => handleDelete(facetValue)}
          >
            {facetNameAndValueString(facetValue)}
            <FontAwesomeIcon icon={faTimes} className="ms-2"/>
          </button>
        )
      })}
    </div>
  )
}

export default SearchCriteriaView
```